src/test/java/org/folio/rest/impl/PurchaseOrderLineNumberTest.java

```diff
@@ -120,10 +122,10 @@ private int retrievePoLineNumber(String purchaseOrderId) throws MalformedURLExce
       .path("sequenceNumber"));
   }
 
-  private static ResultSet execute(String query) {
+  private static RowSet execute(String query) {
```

Comment:
RowSet can be parametrized as RowSet<Row>

Reply:
Done